index.html

```diff
@@ -129,6 +146,17 @@ <h3 class="feature-title">Fast-paced Courses</h3>
 
   <div class="container overflow-hidden text-center">
     <div class="row gx-6">
+      <div class="col">

```

Comment:
The Image of the Python is not showing ? May be the path of the image is not correct or may be the image is not added in the images folder

Reply:
ok sir now you tell me it works So I definetly work on darkmode